src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -695,7 +695,7 @@ export default class ChargingStationService {
 
   public static async handleDeleteChargingStation(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
     // Filter
-    const chargingStationID = ChargingStationSecurity.filterChargingStationRequestByID(req.query);
+    const chargingStationID = ChargingStationValidator.getInstance().validateChargingStationDeleteReq(req.params).id;
```

Comment:
In the router we have in the delete:
`req.query.ID = req.params.id;`
Why do we have a schema here with the 'id' in lowercase?

Reply:
As single ids won't be passed as query strings anymore, I was planning to use lowercase everywhere. What's your point of view ?

Reply:
Before you plan anything, you have to discuss it with me first ;-).
As I wrote, in the router, you are assigning the param to the query in uppercase (req.query.ID = req.params.id) so why it ends up with an 'id' after applying the Json schema?
This is not what has been done in the Tenant schema in similar situation.
Will we have a regression in Angular or in the mobile app if you change this?

Reply:
The `id` parameter passed in path is lowercased. I used `req.query.ID = req.params.id` because security classes require an uppercased `ID` parameter coming from query object (as it was passed as a query parameter before). I made this translation to avoid big refactoring on classes destined to be deleted soon.
Now that we are slowly removing security classes, we don't need to do this "translation" anymore and can use params as they are.
This won't have any effect on Angular/Mobile app, as the HTTP call stay the same.

Reply:
> Before you plan anything, you have to discuss it with me first ;-).

Ok, my bad

Yes you're right, we cannot change parameters as old endpoints with old parameters are still used.
I will set back the req.query.ID = req.params.id;

To explain why I did this (too early) :
The id parameter passed in path is lowercased. I used req.query.ID = req.params.id because security classes require an uppercased ID parameter coming from query object (as it was passed as a query parameter before). I made this translation to avoid big refactoring on classes destined to be deleted soon.
Now that we are slowly removing security classes, we don't need to do this "translation" anymore and can use params as they are.

Reply:
Done, rolled back IDs in query.